html-aam/index.html

```diff
@@ -10958,7 +10958,10 @@ <h4 id="att-high">`high`</h4>
             <tr>
               <th><a href="https://developer.apple.com/reference/appkit/nsaccessibility">AX</a></th>
               <td>
-                <div class="general">Not mapped</div>
+                <div class="general">
+                  Expose "optimal value" via `AXValueDescription` if &lt;value&gt; is greater than or equal to &lt;high&gt;. Expose "suboptimal value" if &lt;value&gt; is less than &lt;high&gt; and
```

Comment:
[@rahimabdi 's migrated comment](https://github.com/w3c/html-aam/pull/538/files#r1585784860):

Relevant WebKit code: https://github.com/WebKit/WebKit/blob/c330ea99229f6bb478dde74497ee652e44a3d7d5/Source/WebCore/html/HTMLMeterElement.cpp#L172

This looks correct for high. We'll also need to add the case for when the value is less than "low" (for which this is exposed as "critical value").